# HG changeset patch
# User Rik <rik@octave.org>
# Date 1479924034 28800
#      Wed Nov 23 10:00:34 2016 -0800
# Branch stable
# Node ID 9d7026337bea5d691f35df20aae3553f14c9c191
# Parent  9ec894c880778f042cf3f57608cca769aeeef47b
axis.m: Replace call to non-existent function clim with caxis.

* axis.m: Replace call to non-existent function clim with caxis.

diff --git a/scripts/plot/appearance/axis.m b/scripts/plot/appearance/axis.m
--- a/scripts/plot/appearance/axis.m
+++ b/scripts/plot/appearance/axis.m
@@ -321,17 +321,17 @@ function limits = __axis__ (ca, ax, vara
       ylim (ca, ax(3:4));
     endif
 
     if (len > 5)
       zlim (ca, ax(5:6));
     endif
 
     if (len > 7)
-      clim (ca, ax(7:8));
+      caxis (ca, ax(7:8));
     endif
 
   else
     error ("axis: expecting no args, or a numeric vector with 2, 4, 6, or 8 elements");
   endif
 
   if (! isempty (varargin))
     __axis__ (ca, varargin{:});
